content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/README.md

```diff
@@ -94,6 +105,11 @@ where `<embeddableResourceType>` is the resource type of your custom embeddable.
 * `./youtubeVideoId` - defines the YouTube video ID.
 * `./youtubeWidth` - defines the YouTube video player width.
 * `./youtubeHeight` - defines the YouTube video player height.
+* `./youtubeMute` - the value of the [YouTube mute parameter][yt-parameters]
```

Comment:
I would prefer these are documented in `apps/core/wcm/components/embed/v1/embed/embeddable/youtube/README.md` as they are specific to this embeddable.